core/storage/suggestion/gae_models.py

```diff
@@ -400,6 +400,42 @@ def get_translation_suggestions_waiting_longest_for_review_per_lang(
             .fetch(MAX_TRANSLATION_SUGGESTIONS_TO_FETCH_FOR_REVIEWER_EMAILS)
         )
 
+    @classmethod
+    def get_number_of_translation_suggestions_in_review_in_lang_code(
+            cls, language_code):
+        """Returns the total number of translation suggestions in the
+        given language code that are in review.
+
+        Args:
+            language_code: str. The language code of the translation
+                suggestions.
+
+        Returns:
+            Int. The total number of translation suggestions in the
```

Comment:
Int --> int, ditto below and elsewhere in this PR.

(If you're copying this from Docs, be careful :) )